provider/data/experimental/fingerprints.csv

```diff
@@ -29347,15 +29347,14 @@ currency/extended@1, zu/YER, 68B, 52c541c97403eaa6
 currency/extended@1, zu/ZAR, 75B, 854159a2eeed4880
 currency/extended@1, zu/ZMK, 85B, f9b17cba085ca192
 currency/extended@1, zu/ZMW, 71B, 608ab48db3a48aad
-currency/patterns@1, <lookup>, 94B, 14 identifiers
-currency/patterns@1, <total>, 617B, 9 unique payloads
+currency/patterns@1, <lookup>, 91B, 13 identifiers
+currency/patterns@1, <total>, 569B, 8 unique payloads
 currency/patterns@1, blo, 82B, da128faf13fad0d5
 currency/patterns@1, ceb, 72B, db7c7c84fecaf3d0
 currency/patterns@1, es-GT, 72B, -> ceb
 currency/patterns@1, ja, 61B, 9af35aad6ebc7d70
 currency/patterns@1, my, 62B, 86cee97085536d8b
 currency/patterns@1, ro, 85B, 629b2b7dcea97875
-currency/patterns@1, sd, 48B, 3a6d9523170345d0
```

Comment:
this seems to be a CLDR bug (missing `Other`), can you file an issue?

Reply:
I have already filed this issue: https://github.com/unicode-org/icu4x/issues/5374. I want to discuss it with @sffc today to ensure that it is a CLDR issue before filing it. If it is a CLDR issue, we need to decide what to do in the meantime. We can use the default `{0} {1}` or the pattern from `en-US`.

Reply:
done